examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<ul><li>I suggest to start with the simple and common example - using the secrets through a function context, then it makes more sense to give the other option, directly through the proejct (and say "also allow")</li><li>I think the "Client operations are..." is kind of a design explanation and is not relevant here</li><li>The <code>get_secret</code> runs on server side now not ?</li><li>I'm a bit nudnik now, but again I'll go with the basic usage first, i.e. <code>proj.with_secrets('vault', ['aws_key'])</code>, giving empty list to get all secrets is the advanced option (so I would probably not use it, just having the comment explaining it), but this totally a matter of opinion</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
ok